app/src/main/java/com/boolder/boolder/data/database/DatabaseModules.kt

```diff
@@ -22,3 +23,18 @@ val databaseModule = module {
     single { ProblemRepository(get<BoolderAppDatabase>().problemDao()) }
 
 }
+
+val tickDatabaseModule = module {
+    single {
+        Room.databaseBuilder(
+            androidContext(),
+            TickDatabase::class.java,
+            "tick.db"
+        )
+            .fallbackToDestructiveMigration()
```

Comment:
⚠️ This line will make any users losing their tick list upon any migration of the `TickDatabase`

Reply:
Good catch. Removed the line. I guess it does not matter for the database of the problems because it it static?